ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_access_control_record.py

```diff
@@ -66,6 +66,8 @@
 extends_documentation_fragment: hpe.nimble.hpe_nimble
 short_description: Manage the HPE Nimble Storage access control records
 version_added: "1.0.0"
+notes:
+  -  check_mode is not supported.
```

Comment:
Everyone has double space between "-" and "check_mode".

Also, is backticks supported here, as it's a parameter.

Reply:
ok changed to "- This module does not supports C(check_mode)."